v2/api-validator/src/config/index.ts

```diff
@@ -189,12 +189,12 @@ function isStringArray(value: string[]) {
   return true;
 }
 
-function capabilitiesComponent(value: string) {
-  if (value === undefined) {
+function capabilitiesComponent(value: unknown) {
+  if (value == undefined) {
```

Comment:
Stricly check for undefined as the capabilities config is compared with the server response and null is an actual value that we do not expect